<launch>
<node name="server" type="server.py" pkg="imu_driver"/>
<arg name="portimu" default="/dev/ttyUSB0" />
<arg name="portecho" default="/dev/ttyUSB1" />
<node name="echo_driver" pkg="imu_driver" type="echo_location.py" output="screen"/>
<param name="imu_port" value="$(arg portimu)"/>
<param name="echo_port" value="$(arg portecho)"/>
<!-- <node name= "listener" type="listener.py" pkg= "imu_driver"/> -->
<node name="tf_visualizer" type="tf_visualizer.py" pkg = "imu_driver" output="screen" respawn="false"/>
</launch>
